### PR TITLE
Conda

### PR DESCRIPTION
**Pull request by lecorguille**
Hop!
- [x] conda
- [x] planemo test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,53 @@
...
+Travis
+------
+[![Build Status](https://travis-ci.org/workflow4metabolomics/tool-bank-golm-lib_search.svg?branch=master)](https://travis-ci.org/workflow4metabolomics/tool-bank-golm-lib_search)
+
+Test and Deploy with Confidence. Easily sync your GitHub projects with Travis CI and you'll be testing your code in minutes!
+
+Historic contributors
+---------------------
+ - Yann Guitton @yguitton - [LABERCA - Laboratory of Food Contaminants and Residue Analysis](http://www.laberca.org/) - Ecole Nationale Vétérinaire, Agroalimentaire et de l'Alimentation Nantes-Atlantique - France
+ - Gabriel XXXXXXXX - [French Metabolomics and Fluxomics Infrastructure (MetaboHUB)](http://www.metabohub.fr/en) - [La plateforme "Exploration du Métabolisme" (PFEM, Clermont-Ferrand)](http://www6.clermont.inra.fr/plateforme_exploration_metabolisme)
```
Comment (lecorguille) [inline README.md]: @fgiacomoni Can you complete or give me the name of Gabriel for the README

File: .shed.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name: golm_ws_lib_search
+owner: yguitton
```
Comment (lecorguille) [inline .shed.yml:2]: @yguitton @fgiacomoni Who will be the main author?

**Comment by lecorguille** [comment]
Can you also check the  html_output1.html file, if it's correct for you?

If it seems good for you, the next step is to install this version on galaxydev.sb-roscoff.fr
But for that, someone must push it on the testtoolshed.

File: .shed.yml (hunk 2 above)
Comment (fgiacomoni) [inline .shed.yml:2]: My student Gabriel was the main author so I will stay the owner.

**Comment by fgiacomoni** [comment]
I commit your PR - and submit on the testTS and thank you

**Pull request merged**